### ISSUE TITLE
how to reproduce your pre-train model

### ISSUE THREAD

**Issue by shockjiang** [2toinf/X-VLA#10]
Dear authors,
   I'm very glad to find a such an excellent project which provides data as well as sota results. I plan to start my VLA idea based on your project if possible. But one concern raised after reading your readme, that is how to reproduce your pre-train model.  In the train/ft section, you provide a way for ft on a new domain. Is it possiblle to reproduce your pre-train model with the same method?

**Comment by 2toinf** [comment]
Hi～ @shockjiang  Our data reading pipeline and training scripts support both fine-tuning and pre-training reproduction. Please note that you should disable the learning rate scheduler, which is specifically designed for fine-tuning, and ensure that your pre-training data recipe is properly prepared.

**Comment by shockjiang** [comment]
@2toinf thank you for your reply. I'll try to reproduce the pretrain and let u know our result.

**Comment by 307321587** [comment]
> [@2toinf](https://github.com/2toinf) thank you for your reply. I'll try to reproduce the pretrain and let u know our result.谢谢您的回复。我会尝试重新进行预训练，然后告诉您我们的结果。

Hello，have you ever achieve the pretranning? Are all blocks include the VLM block in the network needed to be trained? And the learning rate should be kept in 10-4 in the whole pretrainning process? Thank you very much!
